src/video/x11/SDL_x11window.c

```diff
@@ -1888,6 +1871,39 @@ X11_FlashWindow(_THIS, SDL_Window * window, SDL_FlashOperation operation)
     return 0;
 }
 
+bool SDL_X11_SetWindowTitle(Display* display, Window xwindow, char* title) {
+    Atom _NET_WM_NAME = X11_XInternAtom(display, "_NET_WM_NAME", False);
+    XTextProperty titleprop;
+    int conv = X11_XmbTextListToTextProperty(display, (char**) &title, 1, XStdICCTextStyle, &titleprop);
+    Status status;
+
+    if (conv == 0) {
+        X11_XSetTextProperty(display, xwindow, &titleprop, XA_WM_NAME);
+        X11_XFree(titleprop.value);
+    // we know this can't be a locale error as we checked X locale validity in X11_VideoInit
+    } else if (conv <= 0) {
+        SDL_LogError(SDL_LOG_CATEGORY_VIDEO, "X11 reporting it's out of memory");
+        return EXIT_FAILURE;
```

Comment:
This function should return int and do this instead:
return SDL_OutOfMemory();

Reply:
Darn, should've known about that function. The fact I've made a single SDL patch before this one is showing, ain't it? :sweat_smile: Fixed in 513e0eb.